app/services/session_memory_service.py

```
@@ -133,13 +133,192 @@ def anthropic_client(self):
+                    "expires_at": expires_at.isoformat() if expires_at else None,
+                }
+                if embedding is not None:
+                    retry_payload["embedding"] = embedding
+                await asyncio.to_thread(
+                    lambda: supabase.table("session_memories")
+                    .update(retry_payload)
+                    .eq("id", retry_id)
+                    .execute()
+                )
+                return retry_id
+            raise insert_err
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)